platformio.ini

```diff
@@ -7,3 +7,68 @@
 ;
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+include_dir = firmware/include
+lib_dir = firmware/lib
+src_dir = firmware/src
+test_dir = firmware/test
+
+[env]
+platform = ststm32
+
+# WARNING: For whatever reason, this CANNOT be the
+# disco_b_g431b_esc1 board you probably anticipate
+# it can be. This took my morning don't let it take yours.
+board = genericSTM32G431CB
+framework = arduino
+monitor_dtr = 1
```

Comment:
@runger I'm not 100% sure whether this is needed/helpful.

It was in my config in the past I had working, but it could be incorrect.